components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -3031,6 +3031,9 @@ components:
         displayName:
           type: string
           example: 'My Custom Protocol'
+        configName:
```

Comment:
shall we use name instead of configName? WDYT?

Reply:
There are two different properties as name and configName https://github.com/wso2/carbon-identity-framework/blob/48672a4962308080b3dc3d6a2170fbf837323531/components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/AbstractInboundAuthenticatorConfig.java#L35-L42